substrate/primitives/runtime/src/generic/unchecked_extrinsic.rs

```diff
@@ -245,6 +245,43 @@ pub struct UncheckedExtrinsic<
 	pub preamble: Preamble<Address, Signature, Extension>,
 	/// The function that should be called.
 	pub function: Call,
+	/// Stores the raw encoded call.
+	///
+	/// This is mainly interesting if this extrinsic was created by decoding it from bytes. In this
+	/// case this field should be set to `Some` holding the original bytes used to decode the
+	/// [`Self::function`]. This is done to protect against decode implementations of `Call` that
+	/// are not biderective (encodes to the exact same bytes it was encoded from). If this `field`
```

Comment:
```suggestion
	/// are not bijective (encodes to the exact same bytes it was encoded from). If this `field`
```
Could not find the other word.

Reply:
:see_no_evil: